### DISCUSSION TITLE
Equivalent of an F() query?

### DISCUSSION THREAD

**Discussion by gone** [tortoise/tortoise-orm#878] [Q&A] [answered]
is there some way to copy the django F? EG to do something like:

```
        users.filter(total_characters__lt=F("max_characters")).update(
            ticks_since_character=F("ticks_since_character") + 1
        )
```

**Comment by long2ice** [comment]
Did you see documentation?

**Reply by gone** [reply]
I didn't - I expected it to be grouped with the Q object under "query api" in the docs, but I see now it has it's own section

**Comment by gone** [comment, marked as answer]
Reading the docs a second time it's documented here: https://tortoise-orm.readthedocs.io/en/latest/expressions.html
